cli/internal/terraform/terraform/constellation-cluster/install-constellation.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+if ! command -v constellation &> /dev/null; then
+  OS=$(uname -s)
+  ARCH=$(uname -m)
+
+  URL=""
+
+  if [[ $OS == "Darwin" ]]; then
+    if [[ $ARCH == "arm64" ]]; then
+      URL="https://github.com/edgelesssys/constellation/releases/latest/download/constellation-darwin-arm64"
```

Comment:
you have an idea how to mention the TODO question from above: about how to make this work before we make the next release? (is it good enough to run bazel devbuild in the install script?)

IMO it doesn't really matter since we will publish the module with the release and hence they should coincide. Would just complicate versioning and artifact bundling further.

Reply:
this was decided to be fixed to a specific version, closing.